### ISSUE TITLE
Not able to access instance after adding an item with blank link under Menu

### ISSUE THREAD

**Issue by AlWalid** [huridocs/uwazi#1679]
I was in the process of adding a new page to an Uwazi instance
After creating the page content under pages, I went to Menu to add the page
However, I pressed enter directly which saved a blank link
Following that manipulation, all URL were displaying a blank page
This issue was tested on a second Uwazi instance and produced the same result

**Comment by txau** [comment]
@AlWalid this is already fixed. Will be deployed soon. Wait for testing.

**Comment by AlWalid** [comment]
@txau Thank you for the quick fix !

**Comment by txau** [comment]
@AlWalid its deployed, can you take a look?

**Comment by AlWalid** [comment]
Hi @txau , it was resolved on the 2 instances that I communicated to you after being stuck. But I didn't replicate the bug to see if it is still there.

Now I tried on one of the instance (Testing instance) , and the same issue is happening. After putting a blank link and confirming the instance displays a maintenance message. When it comes back after ~ 1 minute , all pages are blank.

We can make a regex on the link field, having a script testing if the page exists or not, or have a select list, listing all the pages present under pages section and not letting any other input.

Thanks !
